index.html

```diff
@@ -0,0 +1,58 @@
+<!-- DOCTYPE html -->
+<html lang="en-GB">
+	
+	<head>
+		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type="text/css" href="styles.css">
+	</head>
```

Comment:
I think best practice is to add a title tag in the head, after the CSS links:
```suggestion
	<title>Charlotte is brilliant at coding</title>
	</head>
```